__tests__/app.test.js

```diff
@@ -78,6 +78,41 @@ describe("GET: /api/articles/:article_id", () => {
           expect(article).toHaveProperty("article_img_url");
         });
     });
+    test("Should return article 3 with the total comment count on it", () => {
+      test_article_id = 3;
+      return request(app)
+        .get(`/api/articles/${test_article_id}`)
+        .expect(200)
+        .then(({ body }) => {
+          const article = body.article;
+          expect(article.article_id).toBe(3);
+          expect(article).toHaveProperty("comment_count", "2");
```

Comment:
Do we want the comment_count to be 2, or a string of "2"? We'd expect count to be a number, so how can we coerce it?
_Hint_ - this can be done with the SQL CAST() function or you can do it in js 😄